### DISCUSSION TITLE
Dropdown jumps as data is selected

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#1100] [Q&A]
Hi, the dropdown component jumps around quite a bit when the user is selecting choices. It would be good if it stayed in one place... can provide minimal repro if needed, but should be pretty easy to reproduce!

**Comment by Alex-Wenner-FHR** [comment]
*Note, this only happens in smaller view port widths (i.e. Laptop screen, mobile)

**Reply by mturoci** [reply]
@AlexWennerDT can you please make a short video depicting the issue?

**Reply by Alex-Wenner-FHR** [reply]
Hi, I will try to recreate. I attempted with dropdown.py but was unable to recreate this behavior. 

Follow up question, is there a way to enable dropdown as a meta dialog with less than 100 choices? It is a much cleaner look in some circumstances I feel like!

**Reply by mturoci** [reply]
> Follow-up question, is there a way to enable dropdown as a meta dialog with less than 100 choices? It is a much cleaner look in some circumstances I feel like!

Currently, there is none. The only concern I have as usual is misuse - using dialog when not necessary, resulting in poor UX. I Will give it a deeper thought though, thanks for the suggestion @AlexWennerDT! cc @lo5

**Reply by mturoci** [reply]
@AlexWennerDT - https://github.com/h2oai/wave/issues/1116
